op-deployer/pkg/deployer/bootstrap/flags.go

```diff
@@ -180,6 +180,7 @@ var SuperchainFlags = []cli.Flag{
 	deployer.ArtifactsLocatorFlag,
 	SuperchainProxyAdminOwnerFlag,
 	ProtocolVersionsOwnerFlag,
+	L1ProxyAdminOwnerFlag,
```

Comment:
Seems like this flag should be added to `ImplementationsFlags`, not `SuperchainFlags`